src/views/Scenario/Scenario.js

```diff
@@ -68,6 +70,11 @@ const Scenario = (props) => {
     localStorage.setItem('scenarioParametersAccordionExpanded', accordionSummaryExpanded);
   }, [accordionSummaryExpanded]);
 
+  useEffect(() => {
+    if (currentScenario.data === null && sortedScenarioList.length > 0) {
+      setCurrentScenario(sortedScenarioList[0]);
+    }
+  });
```

Comment:
Maybe add currentScenario as argument in UseEffect to avoid execute this at each time a component function is called:

```
useEffect(() => {
    if (currentScenario.data === null && sortedScenarioList.length > 0) {
      setCurrentScenario(sortedScenarioList[0]);
    }
  }, [currentScenario]);
```